### ISSUE TITLE
requirements.txt

### ISSUE THREAD

**Issue by scankannan** [rajikudusadewale/HR-Chatbot#1]
Hi I was curious to test your bot, I notice as per your instructions for testing the chatbot you mention about installing the requirements.txt, as follows

pip install -r requirements.txt

however i do not see that file inside the repository, can you give some directions on where I could find this?

**Comment by rajikudusadewale** [comment]
Sorry about that, it has been added.

**Comment by scankannan** [comment]
thank you and should there be any specic name to the .env file? as when I used the option of "customopenai.env" containing the api key, it generated an erro that i should have the api key called via api or it should be in the .env file.

![image](https://github.com/rajikudusadewale/HR-Chatbot/assets/321902/8dcdc2d3-1b46-44b3-bb9e-40806031bb46)

**Comment by rajikudusadewale** [comment]
env variable wont recognize this [customopenai.env]. 
create a [.env] file. you can create it on VS code studio if that's what you are using or create it as a normal file with the specific name.

then copy and paste this with your own api key:

OPENAI_API_KEY="your api key"

**Comment by scankannan** [comment]
I had created a .env file however as you can see below, it contains the key

![image](https://github.com/rajikudusadewale/HR-Chatbot/assets/321902/420eef09-15db-4d8e-aa81-eaf1fe50e3e0)

I still am getting an error, not sure if you can point me to something that might be wrong here?

**Comment by rajikudusadewale** [comment]
edit to rename the file: remove 'hrchatbot' and leave the file name to '.env'

**Comment by scankannan** [comment]
did that by changing the file name to 'env' as below

![image](https://github.com/rajikudusadewale/HR-Chatbot/assets/321902/90304f57-678a-4b81-bd47-7776758680d4)

the error is still pointing to the OpenAI key which i have already in the env file

**Comment by rajikudusadewale** [comment]
You omitted dot.
Just copy and paste what I typed below:

.env

**Comment by scankannan** [comment]
ooops this was corrected , however the api key gets disabled when I update the .env file, what option should i choose, the email I receive from Open AI

Hi there,

We have determined that your OpenAI API key (sk-pro...SNP) was leaked, and have disabled it with immediate effect.

This may be because you committed your API key to an online service such as GitHub, or your key may have been compromised in another way.

**Comment by rajikudusadewale** [comment]
Yes, You can't commit your private key to Github. Its against their regulations.

You can save the env file on your local/cloud storage to run your chatbot

**Comment by scankannan** [comment]
oh ok got that letme try that

![image](https://github.com/rajikudusadewale/HR-Chatbot/assets/321902/cdc844c0-a5d8-4b9e-90ae-21f194f97087)

**Comment by scankannan** [comment]
however doesnt streamlit look at the github filesystem , I thought it doesnt look at the local filesystem

**Comment by scankannan** [comment]
Hi 

I deleted the .env file from github containing the OpenAI key but have it locally that I have cloned from the Git repository, the file name on my local storage(windows) has the .env file with the Open AI key, but i am not sure if streamlit will be able to refer to the .env file stored locally? Should i have to sync the local storage to the GIT repository? as the Open AI key is not being read for some reason.

![image](https://github.com/rajikudusadewale/HR-Chatbot/assets/321902/e1e61b56-13ed-466a-8deb-b0b4a0564071)

![image](https://github.com/rajikudusadewale/HR-Chatbot/assets/321902/e07c8616-98bb-4d25-a8ee-6c89018ac131)

any thoughts?

**Comment by scankannan** [comment]
Hi raji 

could you help with this?

regards
